Webspage 3.0/styles/main.css

```diff
@@ -0,0 +1,58 @@
+h1 {
+	text-align: left;
+	font-family: roboto;
+	font-variant: regular;
+font-size: 75px;
+font-weight: 50;
+	padding-left: 5%;
+	padding-top: 5%;
+	padding-bottom: 0%;
+	line-height: .5;
+
+}
+
```

Comment:
You will want to add a body style. The body is the full group of visible content on a page. This will prevent you from having to repeat common styles. Then you can just override styles on specific elements. RE:

`body {
  font-family: roboto, sans-serif;
  font-variant: thin;
}`